### ISSUE TITLE
Should docfx work with netcoreapp2.0 projects?

### ISSUE THREAD

**Issue by evil-shrike** [dotnet/docfx#2244]
**DocFX Version Used**:  2.27.0

I haven't found any notes about flavors of .NET that supported so I'm assuming that docfx support .net core projects. But I can't get it working.
When I run `docfx metadata` against some `*.csproj` I'm getting the following:
```
[17-11-29 01:34:11.693]Warning:[ExtractMetadata]Workspace failed with: [Failure] Msbuild failed when processing the file 'D:\Work\R-n-D\XFW3_WebClient\Server.NetCore\WebServer\WebServer.csproj' with message: The SDK 'Microsoft.NET.Sdk.Web' specified could not be found.  D:\Work\R-n-D\XFW3_WebClient\Server.NetCore\WebServer\WebServer.csproj
[17-11-29 01:34:11.903]Warning:[ExtractMetadata]Project 'D:\Work\R-n-D\XFW3_WebClient\Server.NetCore\WebServer\WebServer.csproj' does not contain any documents.
```

While there's a xmldoc file in the project ('Server.NetCore\WebServer\bin\Debug\netcoreapp2.0\Croc.XFW3.Web.xml')
D:\Work\R-n-D\XFW3_WebClient\Server.NetCore\WebServer\WebServer.csproj:
```
<Project Sdk="Microsoft.NET.Sdk.Web">
  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
...
    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
  </PropertyGroup>
```

**Comment by vicancy** [comment]
https://github.com/dotnet/docfx/issues/1752, we are working on resolving it.

**Comment by evil-shrike** [comment]
6 months? 😕

**Comment by vicancy** [comment]
@evil-shrike On-hold for a long time for lack of resource..... Have time working on that now... 😄

**Comment by evil-shrike** [comment]
It's a good news ) . Feel free to close as a dup then. Thnx.

**Comment by vicancy** [comment]
Should be fixed in 2.28.1

**Comment by evil-shrike** [comment]
It's not seem to work, please see #2304 #2303
